### PR TITLE
rds initial

### PR DESCRIPTION
**Pull request by mleager**

### CODE REVIEW

File: level2/asg.tf
Comment (mleager) [inline level2/asg.tf:1]: Initally I was trying to output the RDS's security group to be used the the ASG's security group, and vice versa. Of course this lead to a circular import error, so I only output the RDS's security group to link to the ASG's security group. The ASG security group egress was left open.

File: modules/asg/security_group.tf [hunk 1]
```diff
@@ -18,6 +18,14 @@ resource "aws_security_group" "private" {
     cidr_blocks = ["0.0.0.0/0"]
   }
 
+  # egress {
```
Comment (geoffrey1330) [inline modules/asg/security_group.tf]: could you please, remove commented newline

File: modules/rds/variables.tf [hunk 2]
```diff
@@ -0,0 +1,11 @@
+variable "env_code" {}
+
+variable "vpc_id" {}
+
+variable "private_subnet_id" {}
+
+variable "asg_sg" {}
+
+# variable "secret_value" {
```
Comment (geoffrey1330) [inline modules/rds/variables.tf]: could you please, remove commented newline

File: modules/rds/outputs.tf [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+output "rds_hostname" {
+  value = aws_db_instance.mysql.address
+}
+
+output "rds_username" {
+  value = aws_db_instance.mysql.username
+}
+
+# output "secret" {
```
Comment (geoffrey1330) [inline modules/rds/outputs.tf]: could you please, remove commented newline

File: modules/rds/db.tf [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+  ingress {
+    description     = "Allow Incoming Traffic to MySQL"
+    from_port       = 3306
+    to_port         = 3306
+    protocol        = "tcp"
+    security_groups = [var.asg_sg]
+  }
+
+#   egress {
```
Comment (geoffrey1330) [inline modules/rds/db.tf]: could you please, remove commented newline

**Review by geoffrey1330 — CHANGES_REQUESTED** [review]

**Review by geoffrey1330 — APPROVED** [review]

**Pull request merged**